From 5aa1df94894ab1561a1cee6c47d3c30b84c86a69 Mon Sep 17 00:00:00 2001
From: Kisaragi Hiu <mail@kisaragi-hiu.com>
Date: Sat, 25 Jan 2025 00:08:58 +0900
Subject: [PATCH] Update Japanese translation

Trying my best to match Japanese KDE.

- Update new strings with lupdate.
  ...except the (Wayland) string is left as-is without changing its context.
- Try my best to translate the new strings.
- Use "Caps Lock" for "Caps Lock". This is existing convention, already
  used in eg. kdeplasma-addons, plasma-desktop, and plasma5support.
---
 data/translations/ja.ts | 50 ++++++++++++++++++++++++++++++++++++++---
 1 file changed, 47 insertions(+), 3 deletions(-)

diff --git a/data/translations/ja.ts b/data/translations/ja.ts
index 2bd0bc0b6..d5c178782 100644
--- a/data/translations/ja.ts
+++ b/data/translations/ja.ts
@@ -5,14 +5,14 @@
     <name>PictureBox</name>
     <message>
         <source>Press to login</source>
-        <translation type="unfinished"></translation>
+        <translation>押してログイン</translation>
     </message>
 </context>
 <context>
     <name>QObject</name>
     <message>
         <source>%1 (Wayland)</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 (Wayland)</translation>
     </message>
 </context>
 <context>
@@ -23,7 +23,7 @@
     </message>
     <message>
         <source>Warning, Caps Lock is ON!</source>
-        <translation>注意: Caps ロックがオンになっています!</translation>
+        <translation>注意: Caps Lock が有効です!</translation>
     </message>
     <message>
         <source>Layout</source>
@@ -69,5 +69,49 @@
         <source>Select your user and enter password</source>
         <translation>ユーザ名を選び、パスワードを入力してください</translation>
     </message>
+    <message>
+        <source>Please enter a password!</source>
+        <translation>パスワードを入力してください!</translation>
+    </message>
+    <message>
+        <source>Change password</source>
+        <translation>パスワードを変更</translation>
+    </message>
+    <message>
+        <source>Enter your username</source>
+        <translation>ユーザ名を入力</translation>
+    </message>
+    <message>
+        <source>Enter your password</source>
+        <translation>パスワードを入力</translation>
+    </message>
+    <message>
+        <source>Password:</source>
+        <translation>パスワード:</translation>
+    </message>
+    <message>
+        <source>Show password</source>
+        <translation>パスワードを表示する</translation>
+    </message>
+    <message>
+        <source>Hide password</source>
+        <translation>パスワードを隠す</translation>
+    </message>
+    <message>
+        <source>Suspend</source>
+        <translation>サスペンド</translation>
+    </message>
+    <message>
+        <source>Hibernate</source>
+        <translation>ハイバネート</translation>
+    </message>
+    <message>
+        <source>Password change aborted because maximum tries reached</source>
+        <translation>最大試行回数に達しましたため、パスワード変更は中止しました</translation>
+    </message>
+    <message>
+        <source>New password change round! Please input current password again!</source>
+        <translation>新しいパスワード変更ラウンドです! 今のパスワードをもう一度入力してください!</translation>
+    </message>
 </context>
 </TS>
